packages/void-server/Dockerfile

```diff
@@ -0,0 +1,43 @@
+ARG BASE_IMAGE
+FROM ${BASE_IMAGE} AS builder
+WORKDIR /app
+
+ENV BUILD_PLAYGROUND=1
+
+# process init system
+RUN apt-get update && apt-get install -y --no-install-recommends dumb-init
+RUN npm install pnpm@9.2.0 --global
+RUN pnpm config set store-dir ~/.pnpm-store
+
+# Copy and build the void server package
+COPY pnpm-lock.yaml .
+COPY pnpm-workspace.yaml .
+COPY package.json .
+COPY tsconfig.json .
+COPY turbo.json .
+COPY packages/build-tooling ./packages/build-tooling
+COPY packages/void-server ./packages/void-server
+
+RUN pnpm --filter @scalar/void-server install --frozen-lockfile && \
+    pnpm --filter @scalar/void-server build
```

Comment:
This is good. I think best practice is probably to let base builder build without the flag and then do a second build of the playgrounds here.